MovieApp/MovieApp/App/Features/Movies/ViewControllers/HomeViewController/HomeViewController.swift

```diff
@@ -2,10 +2,24 @@ import UIKit
 
 class HomeViewController: UIViewController {
     
+    let defaultInset = 20
+    let searchBarHeight = 43
+    var searchBarStackView: SearchBarStackView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         buildViews()
+        
```

Comment:
no need for new line

Reply:
Quick recap:
Classes, structs, extensions and all that have a new line at start and at the end.
Every method doesnt have a new line at start and at the end. If you have super. and guard then you need to have a new line after those things.
In +Design files you separate every view with newLine in createViews, styleViews and defineLayoutForViews for visual pleasure.

You are just calling 2 functions inside this block, there is no need to have newLines cos then if you do that all the time it would be hard to read.